src/ext/cache.zsh

```diff
@@ -46,7 +46,7 @@ cat > $ANTIGEN_CACHE <<EOC
 $(functions -- _antigen)
 antigen () {
   local MATCH MBEGIN MEND
-  [[ "\$ZSH_EVAL_CONTEXT" =~ "toplevel:*" || "\$ZSH_EVAL_CONTEXT" =~ "cmdarg:*" ]] && source "$_ANTIGEN_INSTALL_DIR/antigen.zsh" && eval antigen \$@;
```

Comment:
Shouldn't it be `source "$_ANTIGEN_INSTALL_DIR/$_ANTIGEN_INSTALL_FILENAME" ...` ?

Reply:
The code is correct, `${0:A}` will resolve to `/home/$USER/antigen.zsh`